gradle/gradle-mvn-push.gradle

```diff
@@ -1,66 +0,0 @@
-apply plugin: 'com.github.dcendents.android-maven'
-apply plugin: "com.jfrog.bintray"
-
-def siteUrl = 'https://github.com/olivierperez/chucker' // Homepage URL of the library
-def gitUrl = 'https://github.com/olivierperez/chucker.git' // Git repository URL
-group = GROUP // Maven Group ID for the artifact
-
-afterEvaluate { proj ->
-
-    install {
```

Comment:
This block should stay as we need to specify all the parameters for the POM file

Reply:
Ok, got it. In this case I will update fields of the POM file, since some of them are outdated.